src/SIMD.jl

```diff
@@ -560,6 +560,33 @@ end
     end
 end
 
+# Functions taking two arguments, second argument is a scalar
+@generated function llvmwrap(::Type{Val{Op}}, v1::Vec{N,T1},
```

Comment:
You should be able to just use `ccall("@llvm.powi.v4f64", llvmcall, RT, (AT...,), args...)`

Reply:
Is it specific to `@llvm.powi`? Can all other `llvmwrap` methods be implemented with `ccall`?  Maybe only if there is only one instruction other than `declare`?